CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - On X11, add mappings for numpad comma, numpad enter, numlock and pause.
+- Add a value to the `Exit` variant of `ControlFlow` which controls the exit code after running.
```

Comment:
Please prefix these kind of changes with `**Breaking:**` like below

Reply:
Done, is the current version fine?

Reply:
Maybe make it clearer that because of the added constant, this is in practice unlikely to cause breakage?